### ISSUE TITLE
what is the unit of value in .flo

### ISSUE THREAD

**Issue by nerverBeenEasy** [lmb-freiburg/flownet2#66]
Dear flownet authors, I successfully run your project and the performance is outstanding!
But I got little confused about the unit of the two value in .flo. I just know they stand for the  optical flow value in axis x and axis y, but how they quantize to image pixes? Does it mean the number of pixels per second or something else?
Thank you very much!

**Comment by nikolausmayer** [comment]
Hi,

the "optical flow" is defined as the per-pixel motion between the two input images, i.e. if you look up pixel (x,y) in the .flo and get the two values (u,v) there, you should find the pixel at position (x+u,y+v) in the *second* input image.
It does not necessarily make sense to use a time unit here because optical flow can be applied to all sorts of inputs, not just temporally ordered images such as video frames.

TL;DR the unit is just "pixels".

**Comment by nerverBeenEasy** [comment]
Got it, your answer explains well. Thank your for your time!
